src/llv8.cc

```diff
@@ -1047,10 +1097,10 @@ std::string String::Inspect(InspectOptions* options, Error& err) {
   if (err.Fail()) return std::string();
 
   unsigned int len = options->length;
-
+  int total_length = val.length();
   if (len != 0 && val.length() > len) val = val.substr(0, len) + "...";
 
-  return "<String: \"" + val + "\">";
+  return "<String: \"" + val + "\", length=" + std::to_string(total_length) + ">";
```

Comment:
Can you make this a command line option? Or display length before the content like what `JSArray` does.

Reply:
ok